src/components/Garden/Feed/RightPanel.js

```diff
@@ -60,6 +60,26 @@ function RightPanel({
       </div>
     )
   }
+
+  if (mobileMode) {
+    return (
+      <div
+        css={`
+          width: 100%;
+        `}
+      >
+        <HeroBanner onRequestNewProposal={onRequestNewProposal} />
+        {showWrapComponent && (
+          <WrapToken
+            onClaimRewards={onClaimRewards}
+            onUnwrapToken={onUnwrapToken}
+            onWrapToken={onWrapToken}
+          />
+        )}
+      </div>
+    )
+  }
+
```

Comment:
@fabriziovigevani maybe you can help me figure out the right style so in case we are on `largeMode` we display the `Carousel` in a way that we see two buttons at a time. I think this is a good tradeoff because on the `largeMode` we have enough space. In other views, we should show a single button at a time.